patches/patch-sharpziplib-CA63358-TarBuffer.patch

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Cloud Software Group, Inc.
```

Comment:
Minor: I suggest keeping the exact name of the original files (i.e. `patch-sharpziplib-ca-63358-TarBuffer.diff`) so it's easier to diff them.